defaults/config.js

```diff
@@ -139,6 +139,27 @@ module.exports = {
 	// This value is set to `2048` kilobytes by default.
 	prefetchMaxImageSize: 2048,
 
+	// ### `fileUpload`
+	//
+	// Allow uploading files to the server hosting The Lounge.
+	// If enabled, The Lounge will store files in the `${THELOUNGE_HOME}/uploads` folder.
+	//
+	// Files will not expire or be removed by The Lounge, which can cause issues
+	// depending on your hardware, for example in terms of disk usage.
+	//
+	// The available keys for the `fileUpload` object are:
+	//
+	// - `enable`: When `true`, the identd daemon runs on server start.
```

Comment:
@xPaw, I made the following fixes/changes here:

<details>

```diff
@ config.js:145 @ module.exports = {
    // ### `fileUpload`
    //
    // Allow uploading files to the server hosting The Lounge.
-   // If enabled, The Lounge will store files in the `${THELOUNGE_HOME}/uploads` folder.
    //
-   // Files will not expire or be removed by The Lounge, which can cause issues
-   // depending on your hardware, for example in terms of disk usage.
+   // Files are stored in the `${THELOUNGE_HOME}/uploads` folder, do not expire,
+   // and are not removed by The Lounge. This may cause issues depending on your
+   // hardware, for example in terms of disk usage.
    //
    // The available keys for the `fileUpload` object are:
    //
-   // - `enable`: When `true`, the identd daemon runs on server start.
-   // - `maxFileSize`: When file upload is enabled, users sending files above this
-   //    limit will be prompted an error message in their browser.
-   //    Specified value is in kilobytes. Default value is 10000 kilobytes (10 MiB).
-   //
-   // `-1` will disable the file size limit and will allow files of any size. Use at your own risk.
+   // - `enable`: When set to `true`, files can be uploaded on the client with a
+   //   drag-and-drop or using the upload dialog.
+   // - `maxFileSize`: When file upload is enabled, users sending files above
+   //   this limit will be prompted an error message in their browser. A value of
+   //   `-1` disables the file size limit and allows files of any size. **Use at
+   //   your own risk.** This value is set to `10000` kilobytes by default.
    fileUpload: {
        enable: true,
        maxFileSize: 10000,
```
</details>

This is how it looks in the docs page:

<img width="549" alt="screen shot 2018-09-03 at 00 15 46" src="https://user-images.githubusercontent.com/113730/44967723-53cc4180-af11-11e8-822a-4f5a179eedb0.png">

Let me know what you think :)